operators/src/source/gdal_source/loading_info.rs

```diff
@@ -30,18 +30,47 @@ impl MetaData<GdalLoadingInfo, RasterResultDescriptor, RasterQueryRectangle>
         let valid = self.time.unwrap_or_default();
 
         let parts = if valid.intersects(&query.time_interval) {
-            vec![GdalLoadingInfoTemporalSlice {
+            let mut p = vec![];
+
+            if valid.start() > query.time_interval.start() {
+                log::debug!("Missing temporal slice information to provide loading info for the whole query rectangle. Inserting fake first element! Query time: {} First element awailable: {}.", query.time_interval, valid);
+                p.push(GdalLoadingInfoTemporalSlice {
+                    time: TimeInterval::new(query.time_interval.start(), valid.start())
+                        .expect("Condition is checked above"),
+                    params: None,
+                    cache_ttl: self.cache_ttl, // TODO: maybe this needs to be zero.
```

Comment:
why should it be zero? I think it's fine to use the ttl of the data.

Reply:
fill adapter will now take care of ttls